js/components/portal-dashboard/custom-select.tsx

```diff
@@ -65,16 +66,17 @@ export class CustomSelect extends React.PureComponent<IProps, IState> {
 
   private renderList = () => {
     const { items } = this.props;
+    const showWide = this.props.isWide ? css.assignmentList : css.classNav;
```

Comment:
If we assume 200px is the default, I would just do:
`const showWide = this.props.isWide ? css.wide: "";`

Reply:
And then we need the same in the header.